### PR TITLE
issue #214 - link to ISO download page and coreos-installer docs

### PR DESCRIPTION
**Pull request by pjvleeuwen**

### CODE REVIEW

File: modules/ROOT/pages/bare-metal.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ Before installing FCOS, you must have an Ignition configuration file and host it
 
 To install FCOS onto bare metal using the live ISO, follow these steps:
 
-- Download the ISO image:
+- Take the latest ISO image from the https://getfedora.org/coreos/download?tab=metal_virtualized&stream=stable[download page] or using podman (download stable by default, see https://github.com/coreos/coreos-installer/blob/master/docs/cmd/download.md[documentation] for options):
```
Comment (travier) [inline modules/ROOT/pages/bare-metal.adoc:17]: Can you use https://coreos.github.io/coreos-installer/cmd/download/ instead? Thanks!

Comment (pjvleeuwen) [reply modules/ROOT/pages/bare-metal.adoc]: done, thanks for the feedback!

**Review by travier — APPROVED** [review]

File: modules/ROOT/pages/bare-metal.adoc [hunk 2]
```diff
@@ -14,7 +14,7 @@ Before installing FCOS, you must have an Ignition configuration file and host it
 
 To install FCOS onto bare metal using the live ISO, follow these steps:
 
-- Download the ISO image:
+- Take the latest ISO image from the https://getfedora.org/coreos/download?tab=metal_virtualized&stream=stable[download page] or using podman (download stable by default, see https://coreos.github.io/coreos-installer/cmd/download/[documentation] for options):
```
Comment (bgilbert) [inline modules/ROOT/pages/bare-metal.adoc:17]: How about this?

```suggestion
- Download the latest ISO image from the https://getfedora.org/coreos/download?tab=metal_virtualized&stream=stable[download page] or with podman (see https://coreos.github.io/coreos-installer/cmd/download/[documentation] for options):
```

And then let's add `-s stable` to the `podman` options.  Hopefully that'll provide a hint on how to select alternate streams, without emphasizing it too much.

Comment (pjvleeuwen) [reply modules/ROOT/pages/bare-metal.adoc]: better, thanks!

Comment (bgilbert) [reply modules/ROOT/pages/bare-metal.adoc]: Hmm, it looks like you opted not to add `-s stable`?

Comment (pjvleeuwen) [reply modules/ROOT/pages/bare-metal.adoc]: Ah, I considered the update to the CMD documentation as separate. I am not sure how I do that.

Comment (bgilbert) [reply modules/ROOT/pages/bare-metal.adoc]: Oh, I just meant in the example immediately following this line.

Comment (pjvleeuwen) [reply modules/ROOT/pages/bare-metal.adoc]: Ah, now I understand what you meant.

More relevant in this context would be the (also default) `-p metal`.
So I have added them both for consistency with the URL to the download page that I used.

**Review by bgilbert — APPROVED** [review]
Thanks!

**Pull request merged**